client/my-sites/site-settings/form-analytics-stores.jsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import CompactFormToggle from 'components/forms/form-toggle/compact';
+import FormLegend from 'components/forms/form-legend';
+
+class FormAnalyticsStores extends Component {
+	handleToggleChange = ( name ) => () => {
+		this.props.handleToggleChange( name );
+	}
+
+	renderSettings = ( settings, disableAll ) => {
+		const { fields } = this.props;
+
+		return (
+			<div>
+				{
+					settings.map( setting =>
+						<CompactFormToggle
```

Comment:
React gripes about this not having a `key` prop.

> Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of `FormAnalyticsStores`

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/17985/commits/51f29ed52a49f45dc7fd8b45a560a5e05acc8a39